src/main/java/org/jabref/gui/consistency/ConsistencyCheckAction.java

```diff
@@ -0,0 +1,53 @@
+package org.jabref.gui.consistency;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.StateManager;
+import org.jabref.gui.actions.SimpleCommand;
+import org.jabref.gui.preferences.GuiPreferences;
+import org.jabref.logic.quality.consistency.BibliographyConsistencyCheck;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.Field;
+
+public class ConsistencyCheckAction extends SimpleCommand {
+
+    private final DialogService dialogService;
+    private final StateManager stateManager;
+    private final GuiPreferences preferences;
+
+    private final List<String> entryTypes = new ArrayList<>();
+    private final List<String> citationKeys = new ArrayList<>();
+    private final List<String> columns = new ArrayList<>();
+
+    public ConsistencyCheckAction(DialogService dialogService, StateManager stateManager, GuiPreferences preferences) {
+        this.dialogService = dialogService;
+        this.stateManager = stateManager;
+        this.preferences = preferences;
```

Comment:
Add `this.executable.bind(needsDatabase(stateManager));` to ensure the action is disabled when no databse/library is open.